src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -46,6 +47,7 @@
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: INDEX (must be a positive integer) "
             + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_ID + "VOLUNTEERID] "
```

Comment:
Technically volunteer NRIC should not be editable?

Reply:
If its the case where the volunteer manager incorrectly enters the NRIC number, should it still be possible to edit or does the volunteer manager have to delete the whole entry and re-enter the details?

Reply:
Let the manager delete the whole entry and re-enter details. 

Cause if we want to allow them to edit the nric, we have to update all the nric in the record xml to point to the correct volunteer.